lib/Compiler.js

```diff
@@ -480,7 +489,17 @@ class Compiler {
 									: targetFileGeneration + 1;
 							cacheEntry.writtenTo.set(targetPath, newGeneration);
 							this._assetEmittingWrittenFiles.set(targetPath, newGeneration);
-							this.hooks.assetEmitted.callAsync(file, content, callback);
+							this.hooks.assetEmitted.callAsync(
```

Comment:
What about `compilation` here?

Reply:
hm, why not first arg as for many hooks?

Reply:
This hook is more about the emitted asset and compilation is only an extra information